tools/plots.py

```diff
@@ -59,31 +59,44 @@
 parser.add_argument('--plots', type=str,
         default='all',
         help='List of plots to produce (all,')
+parser.add_argument('--platform_file', type=str,
```

Comment:
I didn't even know this file existed, so... @derkling, it's in your hands :p

Reply:
Huh, this is just the platform file. I though that's very basic thing for LISA (EAS needs platform information, etc).

Reply:
I meant the **tools/plots.py** file. I've never used it - @derkling gave me a quick intro to what it is.

Reply:
Ok sounds good, thanks